dashboard/modules/runtime_env/runtime_env_agent.py

```diff
@@ -481,24 +495,30 @@ async def _create_runtime_env_with_retry(
                 else runtime_env_config["setup_timeout_seconds"]
             )
 
+            start = time.perf_counter()
```

Comment:
why `perf_counter` and not `time.time` OOC?

Reply:
No special reasons... I heard perf_counter was a bit more accurate, but I might be wrong. No objection to use time.time if you prefer that.